calibration-app/app/src/main/java/org/dpppt/android/calibration/util/NotificationUtil.java

```diff
@@ -55,6 +59,15 @@ public static void showNotification(Context context, @StringRes int title, @Stri
 
 		NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
 		notificationManager.notify(NOTIFICATION_ID, notification);
+
+		if (AppConfigManager.getInstance(context).getDevHistory()) {
```

Comment:
Is this HistoryDatabase write intentionally done in the calibration app module? Shouldn't this be done in the sdk module?